rholang/src/test/scala/coop/rchain/rholang/interpreter/util/codec/Base58Properties.scala

```diff
@@ -0,0 +1,58 @@
+package coop.rchain.rholang.interpreter.util.codec
+
+import org.scalacheck.{Arbitrary, Gen}
+import org.scalatest.prop.GeneratorDrivenPropertyChecks
+import org.scalatest.{Matchers, PropSpec}
+
+class Base58Properties extends PropSpec with GeneratorDrivenPropertyChecks with Matchers {
```

Comment:
let's have the property config bumped to more trials

Reply:
I managed to get:
```

NumberFormatException was thrown during property evaluation.
  Message: Zero length BigInteger
  Occurred when passed generated values (
    arg0 = (,0, ) // 9 shrinks
  )
ScalaTestFailureLocation: coop.rchain.rholang.interpreter.util.codec.Base58Properties at (Base58Properties.scala:44)
org.scalatest.exceptions.GeneratorDrivenPropertyCheckFailedException: NumberFormatException was thrown during property evaluation.
  Message: Zero length BigInteger
  Occurred when passed generated values (
    arg0 = (,0, ) // 9 shrinks
  )
```

by bumping the settings to

```scala
  implicit val propertyCheckConfiguration = PropertyCheckConfiguration(minSuccessful = 10000, sizeRange = 200)
```

Reply:
It's reproducible AFAICT

Reply:
fixed